services/core-web/src/components/mine/ExplosivesPermit/MineExplosivesPermitTable.tsx

```diff
@@ -223,158 +229,186 @@ const MineExplosivesPermitTable: FC<RouteComponentProps & MineExplosivesPermitTa
         <div
           title="Detonator Quantity"
           className="underline"
-          onClick={(event) => props.handleOpenViewMagazineModal(event, record, "DET")}
+          onClick={(event) => handleOpenViewMagazineModal(event, record, "DET")}
         >
           {text || "0"} units
         </div>
       ),
       sorter: false,
     },
-
     {
       title: "",
+      dataIndex: "addEditButton",
       key: "addEditButton",
       align: "right",
-      render: (record) => {
+      render: (text, record) => {
```

Comment:
I'd actually _not_ add a dataIndex here. If the data has no property "addEditButton", all it does is add a second redundant parameter to the render function.

Reply:
Ah, good point!